src/redux/actions/experimentSettings/navigateFromProcessingTo.js

```diff
@@ -0,0 +1,10 @@
+import {
+  EXPERIMENT_SETTINGS_NAVIGATE_TO,
```

Comment:
I don't feel like this fits in experiment settings

Reply:
I put it in `experimentSettings` because it's the closest to `changedQCFilters`. I'm not sure where else to put it,

Reply:
I think it fits at the root level of the store, with a new reducer and all that

Reply:
it's about navigation, so not really linked with any of the already existing aprts of the store.